### DISCUSSION TITLE
Taro react版本的瀑布流的使用

### DISCUSSION THREAD

**Discussion by maoyutingMyt** [TaleLin/lin-ui#1458] [求助]
1、把dist文件下面copy到了src 然后再config里配置了
 config = {
    navigationBarTitleText: '首页',
    enablePullDownRefresh: true,
    usingComponents: {
      'l-water-flow': '../../static/dist/water-flow/index',
      'l-demo': '../../components/DemoItem',
    },
  };

<View>
        <l-water-flow generic:l-water-flow-item="l-demo" />
        {/*<l-demo />*/}
      </View>

在onload不走wx.lin.renderWaterFlow(this.state.list, false, () => {
      console.log('渲染成功');
    });
在componentDidMount() {
    console.info(wx.lin);
    console.info(1111111111111111111111111);
    wx.lin.renderWaterFlow(this.state.list, false, () => {
      console.log('渲染成功');
    });
  }走但是页面空白渲染不出来

能提供下Taro react 版本的使用瀑布流的例子吗而且也需要下拉加载

**Comment by wuhongdan** [comment]
已经2025年了，写出来了吗？
